### ISSUE TITLE
Guest Access configuration

### ISSUE THREAD

**Issue by dquagliano** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#701]
Please include a link to this: https://docs.microsoft.com/en-us/microsoftteams/set-up-guests.
If you don't set up correctly the guest access, then domain users won't be able to see each other.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59900927-5e5f-77a4-04be-e05a91330fca
* Version Independent ID: 1e4d6b87-f57c-fa45-1630-3c32b821d93c
* Content: [Communicate with Teams users in another organization](https://docs.microsoft.com/en-us/microsoftteams/let-your-teams-users-communicate-with-other-people)
* Content Source: [Teams/let-your-teams-users-communicate-with-other-people.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/let-your-teams-users-communicate-with-other-people.md)
* Service: **msteams**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by AndreaBarr** [comment]
@dquagliano Thank you for this feedback

We appricate you helping to make the docs better. I am sending this off to the Teams team so it can be updated appropriatly. 

I was able to get this into the backlog for the Teams team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
